TokenAdministrationApi.Tests/IntegrationTests.cs

```diff
@@ -52,5 +46,12 @@ public void BaseTearDown()
             _transaction.Rollback();
             _transaction.Dispose();
         }
+
+        [OneTimeTearDown]
+        public void AfterAllTests()
```

Comment:
Just wondering, the names we give to these tests e.g `BaseSetup`, `OneTimeSetUp`, `AfterAllTests`, etc are these keyword phrases that NUnit knows or just great test descriptions 😄

Reply:
The annotations are key words but the method names can be anything :)